packages/core/src/WalletProvider/types.ts

```diff
@@ -50,7 +50,7 @@ export type NetworkInfo = {
   stake: StakeSummary;
 };
 
-export interface CardanoProvider {
+export interface WalletProvider {
   ledgerTip: () => Promise<Cardano.Tip>;
   networkInfo: () => Promise<NetworkInfo>;
   stakePoolStats?: () => Promise<StakePoolStats>;
```

Comment:
Please add a todo here: move `stakePoolStats` out to other provider type, since it's not required for wallet operation.